frontend/src/plugins/sql-lsp/types/sql.ts

```diff
@@ -0,0 +1 @@
+export type SQLDialect = "mysql" | "postgresql";
```

Comment:
Looks like we already have the `SQLDialect` type definition. https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/frontend/src/types/sqlEditor.ts?L10

Reply:
Yes. But re-defining it is to ensure `/plugins/sql-lsp` won't use any types or codes outside the folder.

Reply:
SG